packages/keyring/src/testing.ts

```diff
@@ -71,8 +94,9 @@ export const PAIRS: PairDef[] = [
  */
 export function createTestKeyring (options: KeyringOptions = {}, isDerived = true): KeyringInstance {
   const keyring = new Keyring(options);
+  const pairs:PairDef[] = (options.type && options.type === 'ethereum') ? PAIRSETHEREUM : PAIRSSR25519;
```

Comment:
```suggestion
  const pairs: PairDef[] = (options.type && options.type === 'ethereum') ? PAIRSETHEREUM : PAIRSSR25519;
```

This issue with this one is that it won't actually ever get used, for instance in the apps UI. The test keyring has no type that is applied.

Reply:
>this one

What are you referring to? The type or PAIRSETHEREUM?